<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestProduct</class>
 <widget class="QDialog" name="RequestProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Request">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>780</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>제품등록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RequestClose">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>780</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Requesthead">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Requestcontent">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>551</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Request</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RequestSymptom">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Product name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Symptom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RequestType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type (ex Labtop)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Etc</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
